CleanCodeApp/Modules/Features/Rum/ContactsUs/RumContactUsViewController.swift

```diff
@@ -8,73 +8,113 @@
+        chatButton.translatesAutoresizingMaskIntoConstraints = false
+        return chatButton
+    }()
+    
+    private lazy var messageLabel: UILabel = {
         let messageLabel = UILabel()
         messageLabel.textColor = .black
         messageLabel.font = UIFont.systemFont(ofSize: 16, weight: .semibold)
         messageLabel.text = "Ou envie uma mensagem"
         messageLabel.numberOfLines = 2
         messageLabel.setContentHuggingPriority(.defaultLow, for: .horizontal)
+        messageLabel.translatesAutoresizingMaskIntoConstraints = false
         
+        return messageLabel
+    }()
+    
+    private lazy var sendMessageButton: UIButton = {
         let sendMessageButton = UIButton()
         sendMessageButton.backgroundColor = .blue
         sendMessageButton.setTitle("  Enviar ", for: .normal)
         sendMessageButton.setTitleColor(.white, for: .normal)
         sendMessageButton.layer.cornerRadius = 10
         sendMessageButton.setContentHuggingPriority(.required, for: .horizontal)
-        sendMessageButton.addTarget(self, action: #selector(messageSend), for: .touchUpInside)
-        
-        textView.text = "Escreva sua mensagem aqui"
+        sendMessageButton.addTarget(self, action: #selector(didTapSendMessageButton), for: .touchUpInside)
+        sendMessageButton.translatesAutoresizingMaskIntoConstraints = false
+        return sendMessageButton
+    }()
+    
+    private lazy var closeButton: UIButton = {
         let closeButton = UIButton()
         closeButton.setTitle("Voltar", for: .normal)
         closeButton.setTitleColor(.blue, for: .normal)
         closeButton.backgroundColor = .clear
         closeButton.layer.borderWidth = 1
         closeButton.layer.borderColor = UIColor.blue.cgColor
         closeButton.layer.cornerRadius = 10
-        closeButton.addTarget(self, action: #selector(close), for: .touchUpInside)
-
-        
+        closeButton.addTarget(self, action: #selector(didTapCloseButton), for: .touchUpInside)
+        closeButton.translatesAutoresizingMaskIntoConstraints = false
+        return closeButton
+    }()
+    
+    // MARK: - Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+        fetchData()
+    }
+    
+    // MARK: - Setup Methods
+    private func setupView() {
+        view.backgroundColor = .systemGray6
+        setupButtonImages()
+        setupHierarchy()
+        setupConstraints()
+    }
+    
+    private func setupButtonImages() {
         let symbolConfiguration = UIImage.SymbolConfiguration(pointSize: 36)
         phoneButton.setImage(UIImage.init(systemName: "phone")?.withConfiguration(symbolConfiguration), for: .normal)
         emailButton.setImage(UIImage.init(systemName: "envelope")?.withConfiguration(symbolConfiguration), for: .normal)
         chatButton.setImage(UIImage.init(systemName: "message")?.withConfiguration(symbolConfiguration), for: .normal)
```

Comment:
por mais que essa funcao esteja pequena, repara que a maior parte da linha tem codigo repetido
aplicar DRY aqui
sugiro fazer uma extension de uibutton que tenha: setImage(UIImage.init(systemName: image)?.withConfiguration(symbolConfiguration), for: .normal)

Reply:
ótima sugestão, extension criada!! :)